src/constants.py

```diff
@@ -10,6 +10,6 @@
 FLASK_ENV_CONFIG_PREFIX = "FLASK_"
 FLASK_DATABASE_NAME = "flask-app"
 FLASK_SUPPORTED_DB_INTERFACES = {"mysql_client": "mysql", "postgresql_client": "postgresql"}
-FLASK_BASE_DIR = pathlib.Path("/srv/flask")
+FLASK_BASE_DIR = pathlib.Path("/flask")
```

Comment:
Constants were much discussed in this PR:
https://github.com/canonical/synapse-operator/pull/36

Should we keep using this file/adding constants here or should FLASK_BASE_DIR be created on webserver.py and then imported on the unit test?

Reply:
I think the `FLASK_*_DIR` is not just used in the `webserver.py`, it's used in the `charm_state.py` and `database_migration.py` as well. That's why I think it can be put in the `constants.py`.